app/models/user/academics/registrations.rb

```diff
@@ -14,11 +14,23 @@ def all
       end
 
       def term_ids
-        all.map { |attr| attr.term_id }.compact.uniq
+        all.map { |reg| reg.term_id }.compact.uniq
       end
 
       def find_by_term_id(term_id)
-        all.select { |attr| attr.term_id == term_id }
+        all.select { |reg| reg.term_id == term_id }
+      end
+
+      def latest_academic_level_descriptions
+        latest.collect {|reg| reg.career_preferred_level.description }
```

Comment:
I know I've been using `Enumerable#map`, do you think should we coalesce around `#collect` because it's the more Ruby-ish name?

Reply:
@developish - I think either is fine, but collect makes more sense if you're collecting some specific thing from a collection of objects. 'map' does seem more abstract

Reply:
Ok, I'll start using `collect` going forward